index.js

```diff
@@ -49,7 +50,10 @@ module.exports = function (options, middleware) {
         next = next || nop;
 
         if (options.breakOnError) {
-            cache.once('error', function (err) {
+            // can't use `once` here because it overwrites previously set listener
```

Comment:
This is not true - https://github.com/joyent/node/blob/master/lib/events.js#L206

`once` is attached through `on` and removed after callback is called - basiclly same as you do below.